scala-package/core/pom.xml

```diff
@@ -138,10 +138,22 @@
       <version>INTERNAL</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
You can remove junit here. It is inherited from the parent pom

Reply:
Not sure where you are seeing JUnit in core/pom.xml.